unit_tests/src/main.cpp

```diff
@@ -1,5 +1,5 @@
 #define CATCH_CONFIG_RUNNER
-#include <catch2/catch.hpp>
+#include <catch2/catch_all.hpp>
```

Comment:
Should `CATCH_CONFIG_RUNNER` still be used?

I don't remember seeing that in any of the other PRs.

Reply:
Yes. It's needed for v2 still.

Reply:
Got it. So some of the OS we support still ship v2?

Reply:
Got it. Resolving.